syntax = "proto3";
package import_proto;

message importedMessage{
  string imported_string = 4;
}